### DISCUSSION TITLE
Require an Integer in Number Field

### DISCUSSION THREAD

**Discussion by gkemp94** [payloadcms/payload#1619] [Feature Requests & Ideas]
**Use Case:** For data that is integer by nature (age, podcast season/episode, quantity of stock etc). 

I've submitted the following PR that adds this option + validation. 
https://github.com/payloadcms/payload/pull/1618

Let me know your thoughts/concerns questions etc. 

Cheers,
George

**Comment by jacobsfletch** [comment]
@gkemp94 looks like that PR got all the answers you needed, but just to reiterate — setting `admin.step: 1`, then adding in your own custom validation function, is exactly what you need. Great discussion!
